Dockerfile

```diff
@@ -1,23 +1,22 @@
 FROM python:3.8-slim as base
 
-FROM base as builder
 # Any python libraries that require system libraries to be installed will likely
 # need the following packages in order to build
-RUN apt-get update && apt-get install -y build-essential git
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y build-essential git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
 ENV CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 
-ARG install_dev_dependencies=true
+FROM base as builder
 
 WORKDIR /app
 
-# Install stac_fastapi.types
 COPY . /app
 
-ENV PATH=$PATH:/install/bin
```

Comment:
/install doesn't seem to be used for anything?

Reply:
It's used as a directory to hold the outputs of `pip install -e` but I don't think its needed.  The upside is it makes the WORKDIR a little cleaner, downside is it makes the Dockerfile a little more complex.  Personally I'm fine removing it.

Reply:
Could you explain to me how it's configured to be used as a directory to hold the pipe install -e outputs?

Reply:
Ah, I think I actually just stumbled upon the example where this dockerfile came from: https://blog.realkinetic.com/building-minimal-docker-containers-for-python-applications-37d0272c52f3

What's missing is that pip isn't being invoked with `--install-option="--prefix=/install"`, so it's not actually being used. (I thought this might be the case, but there's a lot of implicit python magic I don't know about yet)